docs/tutorials/prometheus.ipynb

```diff
@@ -0,0 +1,471 @@
+        "\n",
+        "You can start with \"coredns_dns_request_count_total\" and \"5\" (secs) to create the Dataset below. Since earlier in the tutorial two DNS queries were sent, it is expected that the metrics for \"coredns_dns_request_count_total\" will be \"2.0\" at the end of the time series:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "h21RdP7meGzP",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "from datetime import datetime\n",
+        "import tensorflow_io as tfio\n",
+        "\n",
+        "dataset = tfio.experimental.IODataset.from_prometheus(\n",
+        "    \"coredns_dns_request_count_total\", 5)\n",
+        "\n",
+        "print(\"Dataset Spec:\\n{}\\n\".format(dataset.element_spec))\n",
+        "\n",
+        "print(\"CoreDNS Time Series:\")\n",
+        "for (time, value) in dataset:\n",
+        "  # time is milli second, convert to data time:\n",
+        "  time = datetime.fromtimestamp(time // 1000)\n",
+        "  print(\"{}: {}\".format(time, value['coredns']['localhost:9153']['coredns_dns_request_count_total']))"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "8y-VpwcWNYTF",
+        "colab_type": "text"
+      },
+      "source": [
+        "Further looking into the spec of the Dataset:\n",
+        "```\n",
+        "(\n",
+        "  TensorSpec(shape=(), dtype=tf.int64, name=None),\n",
+        "  {\n",
+        "    'coredns': {\n",
+        "      'localhost:9153': {\n",
+        "        'coredns_dns_request_count_total': TensorSpec(shape=(), dtype=tf.float64, name=None)\n",
+        "      }\n",
+        "    }\n",
+        "  }\n",
+        ")\n",
+        "\n",
+        "```\n",
+        "\n",
+        "It is obvious that the dataset consists of a `(time, values)` tuple where the `values` field is a python dict expanded into:\n",
+        "```\n",
+        "\"job_name\": {\n",
+        "  \"instance_name\": {\n",
+        "    \"metric_name\": value,\n",
+        "  },\n",
+        "}\n",
+        "```\n",
+        "\n",
+        "In the above example, 'coredns' is the job name, 'localhost:9153' is the instance name, and 'coredns_dns_request_count_total' is the metric name. Note that depending on the Prometheus query used, it is possible that multiple jobs/instances/metrics could be returned. This is also the reason why python dict has been used in the structure of the Dataset.\n",
+        "\n",
+        "Take another query \"go_memstats_gc_sys_bytes\" as an example. Since both CoreDNS and Prometheus are written in Golang, \"go_memstats_gc_sys_bytes\" metric is available for both \"coredns\" job and \"prometheus\" job:"
```

Comment:
```suggestion
        "Take another query `\"go_memstats_gc_sys_bytes\"` as an example. Since both CoreDNS and Prometheus are written in Golang, `\"go_memstats_gc_sys_bytes\"` metric is available for both `\"coredns\"` job and `\"prometheus\"` job:"
```